environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
     - conda-forge
     - defaults
 dependencies:
-    - python=3.7
+    - python=3.8
```

Comment:
Why 3.8 here?

Reply:
I like 3.8 because the miniconda installer stll has 3.8 as the highest version. But we could bump it straight to 3.9 or keep it at 3.7. This is just for the devs.

Reply:
It looks OK to me. Please see my another comment.